### PR TITLE
add lhe generic filter

### PR DESCRIPTION
**Pull request by covarell**

### CODE REVIEW

**Comment by covarell** [comment]
+1

**Comment by davidlange6** [comment]
@cmsbuild, please test

File: GeneratorInterface/GenFilters/src/LHEGenericFilter.cc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+   // (e.g. close files, deallocate resources etc.)
+
+}
+
+
+// ------------ method called to skim the data  ------------
+bool LHEGenericFilter::filter(edm::Event& iEvent, const edm::EventSetup& iSetup)
+{
+  edm::Handle< LHEEventProduct > EvtHandle ;
+  iEvent.getByLabel( src_ , EvtHandle ) ;
```
Comment (davidlange6) [inline GeneratorInterface/GenFilters/src/LHEGenericFilter.cc]: @covarell - please change to getByToken and add consumes

**Comment by covarell** [comment]
Done in the last commit (also for PR #10817 )

**Comment by covarell** [comment]
+1

**Pull request merged**